NEWS

```diff
@@ -7,6 +7,11 @@ CHANGES WITH 236 in spe:
           numdummies=0, preventing the kernel from automatically creating
           dummy0. All dummy interfaces must now be explicitly created.
 
+        * Unknown specifiers are now rejected. This applies to units and
+          tmpfiles.d configuration. Any percent characters that are not
+          supposed to be interpreted as the beginning of a specifier should be
+          escaped by doubling ("%%").
```

Comment:
I don't think this paragraph is completely accurate because the code added in 038492aed3e0293f to fix https://github.com/systemd/systemd/issues/6374 hasn't been removed. In addition, the commit message in 038492aed3e0293f seems to indicate that it is expected that something like `size=50%,uid=10000` works although `%,` is not a valid specifier.